tests/phpunit/api/v3/SyntaxConformanceTest.php

```diff
@@ -1261,22 +1261,6 @@ public function testNotImplemented_create($Entity) {
     $this->assertStringContainsString(strtolower("API ($Entity, Create) does not exist"), strtolower($result['error_message']));
   }
 
-  /**
-   * @dataProvider entities
-   * @param $Entity
-   */
-  public function testWithoutParam_create($Entity) {
```

Comment:
@eileenmcnaughton I guess this is just testing that an APIv3 Exception or similar is thrown if no params are passed in right?

Reply:
yeah - I think it would be a mandatory field validation error - perhaps it is of some value - but it just feels like we have too many of these tests now